framework/contrib/hit/parse.cc

```diff
@@ -327,17 +281,15 @@ Node::vecStrVal()
 NodeType
 Node::type()
 {
-#ifdef WASP_ENABLED
-  if (_hnv.type() == wasp::COMMENT)
+  if (_hnv.data() == "# blankline")
```

Comment:
this won't cause problems if the document contains the text `# blankline` ?

Reply:
@brandonlangley

Reply:
@dschwen -

This is a good point.

It won't affect any rendering because that uses `Blank` nodes (derived from `Node`) instead of this `node->type()`.

But this could possibly affect something downstream if it directly relies on `node->type() == NodeType::Blank`.

For example, a walker using `NodeType::Blank` somewhere could maybe be affected by `# blankline` in the input.

So I'm making one more small WASP update for this integration that
 - adds a new `wasp::BLANK_LINE` node type and
 - add a `set_type()` interface to our `NodeView` class

Then I'm going to update `framework/contrib/hit/parse.cc` by 
 - adding `_hnv.set_type(WASP::BLANK_LINE)` to the constructor for blanks and
 - updating this line to be `if (_hnv.type() == wasp::BLANK_LINE)` like the others
 
Then there won't be a scenario where this issue could come into play.

Thanks for noticing this.

Reply:
@dschwen -

Thanks for the feedback here. I just added a [commit](https://github.com/idaholab/moose/pull/25432/commits/dcfafa33b1c1aefc0a78762fc3b1e2939c995bb2) that fixes this so it can't be a possible issue.